pkg/controllers/dynakube/oneagent/oneagent_reconciler.go

```diff
@@ -107,6 +115,15 @@ func (r *Reconciler) Reconcile(ctx context.Context) error {
 		return r.cleanUp(ctx)
 	}
 
+	log.Info("creating Pull Secret")
+
+	err = dtpullsecret.NewReconciler(r.client, r.apiReader, r.dynakube, r.tokens).Reconcile(ctx)
+	if err != nil {
+		log.Error(err, "could not reconcile pull secret")
```

Comment:
if you bubble up the error then don't log it, as it will be logged higher up, and we don't want duplicate error logs